cgi-bin/LJ/Web.pm

```diff
@@ -1997,6 +1997,15 @@ sub entry_form_entry_widget {
 }
 
 
+sub minsec_for_user {
+    my $user = LJ::load_user( shift );
+    if ( ! $user ) {
+        return undef;
+    }
```

Comment:
The preferred Dreamwidth style would be:

```perl
my $user = LJ::load_user( $_[0] )
    or return;
```

Almost always, `if !something` should be `unless something`. Also a single-line if statement should be a postfix conditional `return unless something` (dropping the `undef` which is the default return value).